gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/CSVDataSource.java

```diff
@@ -50,11 +53,30 @@ public CSVDataSource(String csvPath, GradoopFlinkConfig config) {
     super(csvPath, config);
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * graph heads will be disposed at the moment. The following issue attempts to provide
```

Comment:
(1) Upper case letter at the beginning of a sentence.
(2) Update the Class-JavaDoc with the additional graphs.csv file